client/tailwind.config.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
       grey: '#83847a',
       'light-grey': '#cecfcb',
       white: '#ffffff',
-      black: '#000000'
+      black: '#000000',
+      purple: '#671e75'
```

Comment:
Instead of copying the value of `primary` into `purple`, you should really just use `primary` in your code.

Reply:
Whoops, I'm blind thanks